.travis.yml

```
@@ -1,7 +1,8 @@
 language: python
 python:
-  - "2.7"
+  - "2.7.15"
   - "3.6"
+  - "3.7"
 sudo: required
```

To use Python 3.7 on travis Ubuntu Xenial is required. see https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905 for details

```suggestion
dist: xenial
sudo: required
```

---

Thank you @jensens for the hint!